lib/msal-common/src/logger/Logger.ts

```diff
@@ -76,6 +76,25 @@ export class Logger {
         return new Logger({loggerCallback: this.localCallback, piiLoggingEnabled: this.piiLoggingEnabled, logLevel: this.level, correlationId: correlationId || this.correlationId}, packageName, packageVersion);
     }
 
+    /**
+     * Returns the validated confitured log level or the default level if
+     * configured level is invalid
+     */
+    private validateLogLevel(loggerOptions: LoggerOptions): number {
+        // Set default log level
+        let logLevel = LogLevel.Info;
+        const configuredLogLevel = loggerOptions.logLevel;
+        // Check if configured log level is a number
+        if (typeof(configuredLogLevel) === "number") {
+            // Check if configured log level is a supported log level
+            if (Object.keys(LogLevel).indexOf(`${configuredLogLevel}`) !== -1) {
+                logLevel = configuredLogLevel;
+            }
+        }
```

Comment:
This feels a little too complicated, considering in theory we could have this logic for every config enum, which is not preferable. I would be fine with simpler logic to fall back to Info if nothing is passed, and then in `logMessage`, throw an error if an invalid log level is set.

Reply:
Either that or do the validation and throw errors in configuration class, where the other defaults are set.

Reply:
I guess we can just make sure its a number and leave it at that. In the off chance someone inputs a negative log level in config, then they just won't see logs, and if they input a level greater than our current highest, they'll just see all the logs.

So in summary, non-numbers in `logLevel` will default to 2 (Info), and numbers outside the range will either mean no logs (< 0) or all logs (> Trace).